client/components/styles/GlobalStyle.js

```diff
@@ -1,6 +1,12 @@
 import { createGlobalStyle } from 'styled-components';
 
 const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+  }
```

Comment:
I'm always a fan of setting both `margin` and `padding` to 0 at the highest level